packages/odyssey/src/scss/components/_tag.scss

```diff
@@ -0,0 +1,13 @@
+.ods-tag {
+  display: inline-block;
+  margin: 0 $tiny-spacing $tiny-spacing 0;
+  padding: 0 $em-tiny-spacing;
+  border-radius: $base-border-radius;
+  background: cv('gray', '100');
+  color: $text-body;
+}
+
+.ods-tag--list {
+  margin-bottom: -$tiny-spacing;
```

Comment:
Whats the story with this line, negative margin? What purpose does it serve?

Reply:
The tags themselves have a `margin-bottom` to ensure they have breathing room when wrapping, but we don't want their margin to affect the height of the tag list itself. This removes that margin from the bottom "row".